wiki/Article_styling_criteria/en.md

```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - wiki standards
+---
+
+# Article styling criteria
+
+*For news posts, see: [News Styling Criteria](/wiki/News_Styling_Criteria)*
+
+The article styling criteria (ASC) serves as the osu! wiki's enforced styling standards to keep consistency in clarity, formatting, and layout in all articles.
+
+All articles should aim at having proper grammar, correct spelling, and correct information.
+
+If you have any suggestions regarding the article styling criteria, [open an issue on GitHub](https://github.com/ppy/osu-wiki/issues/new).
+
+## Standards
+
+*Notice: [RFC 2119](https://tools.ietf.org/html/rfc2119) is used to describe requirement levels.*
+
+The article styling criteria is split up into two documents:
+
+- [Formatting](Formatting) - includes Markdown and other formatting rules
```

Comment:
May be better to use a colon or just no punctuation there at all? This goes for line 23 as well